src/main/java/com/ecore/roles/web/dto/MembershipDto.java

```diff
@@ -0,0 +1,69 @@
+package com.ecore.roles.web.dto;
+
+import com.ecore.roles.model.Membership;
+import com.ecore.roles.model.Role;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+import static java.util.Optional.ofNullable;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class MembershipDto {
+
+    @JsonProperty
```

Comment:
Add all the JSON properties names in the JsonProperty annotations, in order to don't get a code refactoring in the fields names leading to a mismatch in the API clients.